src/frontend/js/components/SearchFilterGroupModal/index.tsx

```diff
@@ -257,8 +241,13 @@ export const SearchFilterGroupModal = ({ filter }: SearchFilterGroupModalProps)
       <Modal
         appElement={modalExclude}
         bodyOpenClassName="has-search-filter-group-modal"
-        className="search-filter-group-modal modal--stretched"
+        className="search-filter-group-modal"
         isOpen={modalIsOpen}
+        title={
+          <legend className="search-filter-group-modal__form__title">
```

Comment:
`legend` tag cannot be used here. This tag can be only used within a fieldset that is no more the case now.